.gitignore

```diff
@@ -1,5 +1,4 @@
 .DS_Store
 node_modules
-dist
```

Comment:
Why did you do this? `dist` is published on npm, but github repo should be clrean.